components/zenFunctions/init_component.sqf

```diff
@@ -44,4 +44,13 @@ if !(isNil 'zen_custom_modules_fnc_register') then
     ["[CA2] Respawners", "Respawn Random Player (OPFOR)", _spawnerEast] call zen_custom_modules_fnc_register;
     ["[CA2] Respawners", "Respawn Random Player (INDFOR)", _spawnerIndfor] call zen_custom_modules_fnc_register;
     ["[CA2] Respawners", "Respawn Random Player (Civilian)", _spawnerCiv] call zen_custom_modules_fnc_register;
+
```

Comment:
In this framework I have accepted dependencies upon ACE and CBA, any other dependencies are fine to include but must be guarded so if the dependency doesn't exist then we don't throw any errors or ideally perform backup behaviour.

In this case if we remove Kyklop's Jamming some time down the line or we use this on a repo without jamming (WW2?) then this will error out if used.  I'd recommend wrapping this section of code in an if-statement which checks for the existence of some variable provided by KYK and simply skips over these calls if it doesn't exist.

```suggestion
if (IS_TRUE(kyk_MyCoolModIsRealAndExists)) then {
```